File: 231225/점수 비교/score-comparison.py
a = input().split()
b = input().split()

a_math = int(a[0])
a_eng = int(a[1])
b_math = int(b[0])
b_eng = int(b[1])

if a_math > b_math and a_eng > b_eng:
    print(1)
else:
    print(0)

File: 240201/점수 비교/score-comparison.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");

let a = input[0].split(" ");
let b = input[1].split(" ");

let aMath = Number(a[0]);
let aEng = Number(a[1]);

let bMath = Number(b[0]);
let bEng = Number(b[1]);

if (aMath > bMath && aEng > bEng) {
    console.log(1);
} else {
    console.log(0);
}